### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by simonmallet**
"As usual" isn't good enough for first timers like me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ URL to be log in to Moodle without typing username and password.
 
 Using
 -----
-1. Install the plugin as usual.
+1. Install the plugin in (Admin > Plugins > Install plugins).
```
Comment (dmitriim) [inline README.md]: Someone like me never uses GUI for installing plugins :) 
Would be good in this case to have something like "Install the plugin using admin interface (Admin > Plugins > Install plugins) or command line"

File: README.md [hunk 2]
```diff
@@ -160,4 +160,4 @@ echo getloginurl('barrywhite@googlemail.com', 'barry', 'white', 'barrywhite', '1
 
 TODO:
 -----
-1. Implement logout webservice to be able to call it from external application.
\ No newline at end of file
+1. Implement logout webservice to be able to call it from external application.
```
Comment (dmitriim) [inline README.md]: Not sure what was changed here?

**Review by dmitriim — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (simonmallet) [inline README.md]: That part wasn't me. I don't know why it sees it as a change :/

Comment (dmitriim) [inline README.md]: Maybe an extra empty line?

**Comment by dmitriim** [comment]
Will close this one off as there is no much progress from the requester.

**Pull request closed without merging**